libs/pika/timing/include/pika/timing/steady_clock.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include <chrono>
 
-namespace pika { namespace chrono {
+namespace pika::chrono {
     using std::chrono::steady_clock;
 
     class steady_time_point
```

Comment:
It feels like it should be possible to just use `steady_clock::time_point`, but then again there's probably a reason why this exists in the first place. If you don't mind, could you check if that would be possible? This is not a blocker for this PR.